examples/angular/src/pages/ui/components/authenticator/sign-up-with-phone/sign-up-with-phone.component.ts

```diff
@@ -2,19 +2,20 @@ import { Component, NgZone, OnInit } from '@angular/core';
 
 import { Amplify } from 'aws-amplify';
 import awsExports from '@environments/auth-with-phone-number/src/aws-exports';
-Amplify.configure({
-  ...awsExports,
-  auth: {
-    login_mechanisms: ['phone_number'],
-  },
-});
 
 @Component({
   selector: 'sign-up-with-phone',
   templateUrl: 'sign-up-with-phone.component.html',
 })
 export class SignUpWithPhoneComponent implements OnInit {
-  constructor() {}
+  constructor() {
+    Amplify.configure({
+      ...awsExports,
+      auth: {
+        login_mechanisms: ['phone_number'],
```

Comment:
I learned yesterday that this structure looks more like:

```js
/* eslint-disable */
// WARNING: DO NOT EDIT. This file is automatically generated by AWS Amplify. It will be overwritten.

const awsmobile = {
    "aws_project_region": "us-east-1",
    "aws_cognito_identity_pool_id": "us-east-1:ea9c8067-5fdc-4203-b2e7-555960e0df2d",
    "aws_cognito_region": "us-east-1",
    "aws_user_pools_id": "us-east-1_FVBg0Ieqr",
    "aws_user_pools_web_client_id": "2p6k4ofki50lg0196r12sniij9",
    "oauth": {},
    "aws_cognito_login_mechanism": [],
    "aws_cognito_signup_attributes": [
        "PHONE_NUMBER",
        "EMAIL",
        "PREFERRED_USERNAME"
    ],
    "aws_cognito_mfa_configuration": "OFF",
    "aws_cognito_mfa_types": [
        "SMS"
    ],
    "aws_cognito_password_protection_settings": {
        "passwordPolicyMinLength": 8,
        "passwordPolicyCharacters": [
            "REQUIRES_LOWERCASE",
            "REQUIRES_NUMBERS",
            "REQUIRES_SYMBOLS",
            "REQUIRES_UPPERCASE"
        ]
    }
};


export default awsmobile;
```

Nothing to change, but just an FYI for when we start supporting it.

Reply:
Cool!